src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -1245,6 +1245,12 @@ static inline unsigned uct_ib_mlx5_qpc_cs_res(unsigned size, int dc)
                          UCT_IB_MLX5_QPC_CS_RES_DISABLE;
 }
 
+enum {
+    UCT_IB_MLX5_QPC_TIMESTAMP_FORMAT_FREE_RUNNING = 0x0,
```

Comment:
Maybe `TIMESTAMP` -> `TS`? Both to be shorter, and for consistency with the fieldname.